jeffreywallphd/OpenFinAL
# Fall2024 dev combined ba5200 mis4000

Feature: Stock Symbol Comparison with Export Options

Description:
This feature allows users to:
1. Input stock symbols for comparison.
2. View metrics comparison for the entered stock symbols.
3. Export the comparison as a CSV or PDF file.

Changes Made:
1. Implemented stock symbol search functionality.
2. Added comparison table for metrics like price, P/E ratio, and market cap.
3. Integrated export options for CSV and PDF.

How to Test:
Run the application locally.
Enter stock symbols (e.g., AAPL, TSLA, AMZN).
Verify the comparison table displays metrics for the stocks.
Test exporting:
- Click "Export to CSV" to download the comparison as a CSV file.
- Click "Export to PDF" to download the comparison as a PDF file.
